src/metax_api/models/catalog_record.py

```diff
@@ -996,6 +996,13 @@ def catalog_is_pas(self):
     def has_alternate_records(self):
         return bool(self.alternate_record_set)
 
+    def state_is_draft(self, request):
```

Comment:
``state_is_draft`` is not a very good name for this method, as this method should only be used when initially saving the dataset. When/If calling the method somewhere else like ``cr.state_is_draft(request)``, the name is misleading, as the method does not actually return the state of the record. Something like ``save_as_draft`` would be more descriptive.

Additionally, the method name should use a prefix ``_``, which is a python practice for saying "this method is private, it should not be used outside of this class"

Reply:
Ok good point!